### PR TITLE
[zh] Fix some nits in kubectl-plugins.md

### PR DESCRIPTION
**Pull request by zhangguanzhang**
Signed-off-by: zhangguanzhang <zhangguanzhang@qq.com>

/assign @tengqm

### CODE REVIEW

File: content/zh/docs/tasks/extend-kubectl/kubectl-plugins.md [hunk 1]
```diff
@@ -346,7 +346,7 @@ The command from the above example, can be invoked using either a dash (`-`) or
 请注意，在插件文件名中引入下划线并不会阻止我们使用 `kubectl foo_bar` 之类的命令。
 可以使用破折号（`-`）或下划线（`_`）调用上面示例中的命令:
 
-```shell
+```bash
```
Comment (tengqm) [inline content/zh/docs/tasks/extend-kubectl/kubectl-plugins.md:349]: shell 挺好，也一致，没必要改成 bash

File: content/zh/docs/tasks/extend-kubectl/kubectl-plugins.md [hunk 2]
```diff
@@ -126,7 +126,7 @@ example, a plugin wanting to provide a new command `kubectl foo`, would simply b
 -->
 ### 示例插件
 
-```
+```bash
```
Comment (tengqm) [inline content/zh/docs/tasks/extend-kubectl/kubectl-plugins.md:129]: 用 shell 就好了

File: content/zh/docs/tasks/extend-kubectl/kubectl-plugins.md (hunk 1 above)
Comment (zhangguanzhang) [reply content/zh/docs/tasks/extend-kubectl/kubectl-plugins.md:349]: 英文是bash

**Comment by zhangguanzhang** [comment]
改好了

**Pull request merged**